test/e2e/storage/csi_volumes.go

```diff
@@ -84,7 +85,15 @@ var _ = Describe("E2E", func() {
 		testsuites.InitVolumeIOTestSuite,
 		testsuites.InitVolumeModeTestSuite,
 		testsuites.InitVolumesTestSuite,
-		testsuites.InitEphemeralTestSuite,
+	}
+	if k8sVersion := os.Getenv("TEST_KUBERNETES_VERSION"); k8sVersion != "" {
+		var major, minor int
+		_, err := fmt.Sscanf(k8sVersion, "%d.%d", &major, &minor)
+		framework.ExpectNoError(err, "failed to parse kubernetes version %s", k8sVersion)
```

Comment:
This assertion triggers: https://cloudnative-k8sci.southcentralus.cloudapp.azure.com/blue/organizations/jenkins/pmem-csi/detail/PR-455/1/pipeline

Spelling error ("kubernetes").

Reply:
Seems like we cannot write Fail() statement inside `Describe/Context`. So now I changed `framework.Logf`, so that we just log the failure and continue running tests.

Reply:
But why does it fail? It shouldn't.

Reply:
> But why does it fail? It shouldn't.

You mean adding `framework.ExpectNoErro()` should not lead to an assertion _or_ errors while parsing version string should not fail e2e tests as I was trying to do ?.
If ti's about assertion - Unlike `It()` The function body in Describe/Context call will be executed by ginkgo immediately. So calling ExpectNoError(), without setting failure handler leads to the assertion.

Reply:
@pohly How to enable 1.14 testing in CI?

Reply:
The Gomega assertion failed, i.e. `err` was non-nil. Reporting that failure then itself failed and panicked (-> stack trace) instead of recording a test failure

That second part makes sense because there is no running test and failure reporting hasn't been configured yet, but `err` should be nil.

Reply:
Or at least that was my interpretation of the failure report. It looks like I was wrong, as there is no log message in the newer test run.